cdmtaskservice/nersc/manager.py

```diff
@@ -219,8 +219,9 @@ def __init__(
         ) / service_group
         self._scratchdir = Path("cdm_task_service") / service_group
         self._jawscfg = f"jaws_cts_{service_group}.conf"
+        self._relative_refdata_root = Path("cdm_task_service") / service_group
```

Comment:
Is this intentional that `self._relative_refdata_root` identical to `self._scratchdir`?

Reply:
Oh, hah, missed that. I'll put a todo there to merge that next time I make changes to the manager